deploy/supervisor/values.yaml

```diff
@@ -258,3 +258,14 @@ endpoints: { }
 #@schema/validation one_of=["true", "false", True, False]
 #@schema/deprecated "This data value will be removed in a future release"
 deprecated_insecure_accept_external_unencrypted_http_requests: false
+
+#@schema/title "Allowed Ciphers for TLS 1.2"
+#@ allowed_ciphers_for_tls_onedottwo_desc = "An array of the ciphers allowed for TLS 1.2. \
+#@ This list will only further constrain the ciphers that Pinniped will use. \
+#@ Specify only secure cipher names from golang's crypto/tls package. \
```

Comment:
This could use an example of how specify a cipher name to make the allowed input format clear.

It would also be more clear if it explained that the ciphers used will be the intersection of this list and the built-in default list.

It would be nice if it gave a pointer to how to look up the current built-in defaults. I'm not sure how to do that though, aside from maybe saying "go look in these particular source code files" or something like that.

Reply:
done, see next push